src/components/leaderboards/View.vue

```diff
@@ -16,6 +16,8 @@ const updateError = ref('')
 
 const token = useSessionToken()
 
+const frontendUrl = import.meta.env.VITE_FRONTEND_URL
```

Comment:
Couldn't inline this as Vite complains that we need to have the project in module mode